src/test/e2e/29_config_file_test.go

```diff
@@ -139,30 +143,37 @@ func configFileDefaultTests(t *testing.T) {
 	}
 
 	// Test remaining default initializers
-	t.Setenv("ZARF_CONFIG", filepath.Join("src", "test", "zarf-config-test.toml"))
-	defer os.Unsetenv("ZARF_CONFIG")
+	envKey := "ZARF_CONFIG"
```

Comment:
nit: Could just set as `const` mayb?

Reply:
def, opted to keep it consistent but const was preferred